template/src/pages/home/Home.module.css

```diff
@@ -22,6 +22,12 @@
   line-height: 85px;
   text-align: center;
   word-wrap: break-word;
+
+  /* Expected "context" media feature range notation (media-feature-range-notation)Stylelintmedia-feature-range-notation */
+  /* stylelint-disable-next-line media-feature-range-notation */
+  @media (max-width: 768px) {
+    font-size: 64px;
+  }
```

Comment:
What?

https://stylelint.io/user-guide/rules/media-feature-range-notation

Should I be using `>=`?

Reply:
whatever uncle stylelint says, we do. ahahaahah